src/lib.rs

```diff
@@ -398,9 +428,9 @@ impl fmt::Display for PatternError {
 
 /// A compiled Unix shell style pattern.
 ///
-/// `?` matches any single character
+/// `?` matches any single character.
```

Comment:
I'd probably change this list of pattern syntax items to be an actual markdown list :)